sdks/java/core/src/main/java/org/apache/beam/sdk/coders/AvroCoder.java

```diff
@@ -349,11 +327,8 @@ public void verifyDeterministic() throws NonDeterministicException {
 
   /**
    * Returns a new {@link DatumWriter} that can be used to write to an Avro file directly.
-   *
-   * @deprecated For {@code AvroCoder} internal use only.
    */
   // TODO: once we can remove this deprecated function, inline in constructor.
-  @Deprecated
```

Comment:
https://issues.apache.org/jira/browse/BEAM-2077 for followup. Re-deprecated in the meantime.